packages/ui/src/components/kanban/Panel.svelte

```diff
@@ -12,28 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 -->
-
 <script lang="ts">
   import { IntlString } from '@anticrm/status'
-  import { Label, ScrollBox } from '@anticrm/ui'
+
+  import Label from '../Label.svelte'
+  import ScrollBox from '../ScrollBox.svelte'
 
   export let title: IntlString
   export let counter: number | undefined
   export let color: string = '#F28469'
   let collapsed: boolean = false
 </script>
 
-<section class="panel" on:dragover on:drop class:collapsed={collapsed}>
-  <div class="header" style="background-color: {color}" on:click={() => collapsed = !collapsed}>
-    {#if collapsed !== true}<div class="title"><Label label={title}/></div>{/if}
+<section class="panel" on:dragover on:drop class:collapsed>
+  <div
+    class="header"
+    style="background-color: {color}"
```

Comment:
Can you please fix style as figma design, #222 as example

Reply:
@BykhovDenis  I'm not sure it is actual mockup, at least I see multiple versions, interactive one is different: https://www.figma.com/proto/coVb3AHZaVKGERM806zHGv/Platform-Prototype?page-id=0%3A1&node-id=74%3A68315&viewport=-30626%2C-4086%2C0.8550704121589661&scaling=contain

@z2sx @MarinaMatijaca which one should be implemented?
![2021-07-17-194122_371x730_scrot](https://user-images.githubusercontent.com/271486/126037212-1e7dff17-7842-4af1-a6da-40f2bdf36943.png)
or
![2021-07-17-194111_282x641_scrot](https://user-images.githubusercontent.com/271486/126037213-4109a1c4-37aa-4d0d-820b-ed0a8c3d6fca.png)